addon/components/hyper-search.js

```diff
@@ -18,12 +18,14 @@ const {
  * Returns the key for the query in the cache. Only works in conjunction with
  * Ember.get.
  *
+ * Replaces perious (.) with dashes (-)
+ *
  * @public
  * @param {String} query
  * @return {String} nested key name
  */
 function keyForQuery(query) {
-  return `_cache.${query}`;
+  return query.replace('.', '-');
```

Comment:
That was my original thought. I wanted to use the same method in 'removeFromCache' though. Since we have to use plain JS to delete the key, the only other option would be to create another function called 'safeCacheKey' or something like that. 

I'm sure though if 'removeFromCache' is ever called though...so maybe it doesn't matter?